packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -249,10 +271,22 @@ const generateComponentMeta = (): void => {
 
                 let [, variable] = (values.match(/([A-Z_]{5,})/) || []) as [string, string | string[]];
                 if (variable) {
-                  const [, variableImportPath] = source.match(new RegExp(`${variable}[\\s\\S]+?from '(.+)';`)) || [];
+                  let variableImportPath;
+                  if (
+                    (typeof variable === 'string' && source.includes(variable)) ||
+                    (typeof variable !== 'string' && variable.some((v) => source.includes(v)))
+                  ) {
+                    [, variableImportPath] = source.match(new RegExp(`${variable}[\\s\\S]+?from '(.+)';`));
+                  } else {
+                    [, variableImportPath] = sharedPropsExportFile.match(
```

Comment:
Why would this be needed? Before we replaced a potentially spread  `...variable` 
 in `rawPropTypes` with its actual values. `propTypes` is based on it and should therefore be clean and contain everything we need.

Reply:
Correct, but but the single importpaths of the shared props can't be found in the component anymore. The single importpaths can only be found in the file, where the shared props are defined :/